client/src/components/Form/index.js

```diff
@@ -68,9 +68,16 @@ class MainForm extends Component {
     this.setState({dateFieldsValues: newFieldsValues});
   }
 
+  onFormSubmit = (values) => {
+    //console.log("Values submitted ::", values);
```

Comment:
Please delete this console log and add it to the issue with an explanation if necessary.